get-started/part2.md

```diff
@@ -139,14 +146,14 @@ the error message.
 > **Note**: Accessing the name of the host when inside a container retrieves the
 container ID, which is like the process ID for a running executable.
 
-## Build the app
-
 That's it! You don't need Python or anything in `requirements.txt` on your
 system, nor will building or running this image install them on your system. It
 doesn't seem like you've really set up an environment with Python and Flask, but
 you have.
 
-Here's what `ls` should show:
+## Build the app
+
+We are ready to build the app. Make sure you are still at the top level of your new project directory. Here's what `ls` should show:
```

Comment:
Again I wouldn't say "project directory" since that's an overloaded term (one that has specific meanings in IDEs, text editors, and so forth). It sounds like we expect them to create something special in an IDE; we just need a bare folder with three files in it.